.ci/jobs.t/defaults.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+##### GLOBAL METADATA
+
+- meta:
+    cluster: elasticsearch-ci
+
+##### JOB DEFAULTS
+
+- job:
+    vault:
+      url: https://secrets.elastic.co:8200
+      role_id: 1ba1ac3e-aee4-d040-d9a3-6ae23bd2b3db
+    node: "general-purpose"
+    concurrent: true
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 90
+      artifactDaysToKeep: 7
+    scm:
+      - git:
+          name: origin
+          # master node jenkins user ~/.ssh
+          credentials-id: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+          reference-repo: "/var/lib/jenkins/.git-references/elasticsearch.git"
+          branches:
+            - "%BRANCH%"
+          url: "https://github.com/elastic/elasticsearch.git"
+          basedir: ""
+          wipe-workspace: "yes"
+    wrappers:
+      - timeout:
+          type: absolute
+          timeout: "120"
+          fail: true
+      - ansicolor
+      - timestamps
+      # TODO: No support un JJBB ?
```

Comment:
We had to use a custom JJB module for this one. I'm not sure how that translate to JJBB. We need to sort this out though as this is how we gather published build scans from Jenkins. Without this HOMER won't be able to find build scans either so this is a show stopper IMO unless we can sort it out.

Reply:
I was planing on sorting this out before I remove the job from the .ci repo. 
Right now it's ok to be missing the notifications because this will be an additional job, the purpose is to make sure all the mechanics of creating and updating the jobs work.